scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -565,19 +565,28 @@
                  mem {50 ..., 95 ..., 100 ...}}
    :totals {mem ..., cpus ..., ...}}"
   [resource-maps]
-  {:percentiles (pc/map-from-keys
-                  (fn percentiles
-                    [resource]
-                    (let [resource-values (->> resource-maps
-                                               (map #(get % resource))
-                                               (remove nil?))]
-                      (-> resource-values
-                          (task-stats/percentiles 50 95 100)
-                          tools/format-resource-map)))
-                  ["cpus" "mem"])
-   :totals (->> resource-maps
-                (reduce (partial merge-with +))
-                tools/format-resource-map)})
+  (let [resources-of-interest ["cpus" "mem"]]
+    {:largest-by (pc/map-from-keys
```

Comment:
How does this differ from the 100'th percentile reported below?

Reply:
Added an explanatory comment